site/pages/docs/ref/filter/filter-en.hbs

```diff
@@ -159,6 +169,24 @@
 					</dl>
 				</td>
 			</tr>
+			<tr>
+                <td><code>filterType</code></td>
+                <td>Sets the type of search operator that will be applied to the filter. By default the filter will search for an exact pattern match, however developers can set the filter to search using an OR or AND operator.</td>
+                <td><code>data-wb-filter='{"filterType": "<em>[Type of Search]</em>"}'</code></td>
+                <td>
+                    <dl>
+                        <dt>None (default):</dt>
+                        <dd>Will use the default exact pattern match when searching for items</dd>
+                        <dt>Type of Search</dt>
+                        <dd>
+							<ul class="list-unstyled">
+								<li><p><code>and</code></p><p>Will use an AND operator between search terms. Only items that contain all search terms will be shown</p></li>
+								<li><p><code>or</code></p><p>Will use an OR operator between search terms. All items that contain any of the supplied search terms will be shown</p></li>
+							</ul>
```

Comment:
IMO it seems kind of strange to have a "Type of Search" term that describes 2 options while leaving the third option (none) as a separate term.

Consider removing "Type of Search", removing its hidden list and placing ``and``/``or`` in the description list itself. Kind of like what's done in the [tabbed interface's documentation page](http://wet-boew.github.io/v4.0-ci/unmin/docs/ref/tabs/tabs-en.html#config).